homepage/_packages/cloud2edge/tour.md

```diff
@@ -488,6 +694,20 @@ In order to add more twins, we simply create additional devices via ["Register a
 and add twins for them with the above snippet by simply adjusting the *device id* and *thing id* in the URL of both
 HTTP requests.
 
+#### Publish telemetry data
+
+To send a telemetry message via Hono's HTTP protocol adapter and thereby update the device's twin representation, the
+following command can be used:
+
+{% clipboard %}
+curl -i -k -u my-auth-id-1@my-tenant:my-password -H 'application/json' --data-binary '{
+  "topic": "org.acme/my-device-1/things/twin/commands/modify",
+  "headers": {},
+  "path": "/features/temperature/properties/value",
+  "value": 45
+}' ${HTTP_ADAPTER_BASE_URL:?}/telemetry
+{% endclipboard %}
```

Comment:
Having a closer look in the Jaeger Tracing UI about how this telemetry request is processed, I see an error.

![send_telemetry__wrong_connection](https://github.com/eclipse/packages/assets/891931/218144f1-3995-4095-9b7a-174494ab72e8)
In the `kafka_consume` step, there is still the right Ditto connection. 
But in the `connectivity_message_mapping` step, the connection that was created for the cloud2edge default `org.eclipse.packages.c2e` tenant is used - so the wrong connection.
Following that, there is a 404 error trying to find the device in this `org.eclipse.packages.c2e` tenant.

Strangely, when I had first created this twin _without a policy_ (as a workaround for the policy creation error - see my other comment), the same telemetry request didn't show this wrong connection usage.

@thjaeckle Do you have an idea here?

---
To enable tracing, I used these chart parameters:
```
hono:
  jaegerBackendExample:
    enabled: true
ditto:
  global:
    tracing:
      enabled: true
      otelExporterOtlpEndpoint: "http://c2e-jaeger-collector:4317"
      sampler: always
```
The URL of the Jaeger UI is in the `JAEGER_QUERY_BASE_URL` env variable set by the `setCloud2EdgeEnv.sh` script.

Reply:
That is expected if both connections use the same authorizationContext.
A connection target will not be notified about events which were the result of a command consumed via the source of the same connection.

However if a second connection exists which has the permissions to read the updated thing, then this connection will emit the event to the configured target

Each connection for different Hono tenants should use a separate authorizationContext string to prevent that.

Reply:
So, I just replace all occurrences of `pre-authenticated:hono-connection` with `pre-authenticated:hono-connection-my-tenant` in the connection JSON and the JSON of the policy concerning that tenant?

Reply:
Yes - if those are 2 separate tenants they should use different "pre authentication" in connections and in their policies.

Reply:
Thanks, I've made that change and it's working now.